changelogs/fragments/89-dynamic-user-password-in-client-my-cnf-and-correct-package-name.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - role_proxysql - Dynamic user/password in .my.cnf
+  - role_proxysql - Correct package name (python3-mysqldb instead of python-mysqldb)
```

Comment:
```suggestion
  - role_proxysql - Dynamic user/password in .my.cnf (https://github.com/ansible-collections/community.proxysql/pull/89).
  - role_proxysql - Correct package name (python3-mysqldb instead of python-mysqldb) (https://github.com/ansible-collections/community.proxysql/pull/89).
```